components/provisioning/org.wso2.carbon.identity.provisioning/src/main/java/org/wso2/carbon/identity/provisioning/OutboundProvisioningManager.java

```diff
@@ -565,11 +564,7 @@ public void provision(ProvisioningEntity provisioningEntity, String serviceProvi
                         boolean isBlocking = entry.getValue().isBlocking();
                         boolean isPolicyEnabled = entry.getValue().isPolicyEnabled();
                         if (isPolicyEnabled) {
-                            isAllowed = XACMLBasedRuleHandler.getInstance().isAllowedToProvision(spTenantDomainName,
-                                    provisioningEntity,
-                                    serviceProvider,
-                                    idPName,
-                                    connectorType);
+                            // engage policy evaluation here
```

Comment:
Format comments.
Comments needs to be started with Capital letter and ends with fullstop.

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/4471